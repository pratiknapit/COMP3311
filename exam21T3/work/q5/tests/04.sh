python3 ./q5 Apartment 1500000 3 2 1
