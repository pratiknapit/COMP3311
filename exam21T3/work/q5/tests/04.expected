#45760: 7/95 Jacques Street, Kingsford 2032, 3br, 2ba, 1car, $888000
#46823: 13/80 Smith Street, Kingsford 2032, 3br, 2ba, 1car, $1120000
#46113: 11/69 Brennan Street, Alexandria 2015, 3br, 2ba, 1car, $1163000
#45702: 7/90 Allen Avenue, Alexandria 2015, 3br, 2ba, 1car, $1281000
#46025: 5/59 Loveridge Street, Alexandria 2015, 3br, 2ba, 1car, $1284000
#45699: 5/27 Leeton Avenue, Coogee 2034, 3br, 2ba, 1car, $1474000
